diff --git a/node_modules/@vue/reactivity/dist/reactivity.d.ts b/node_modules/@vue/reactivity/dist/reactivity.d.ts
index 4b00720..04405f9 100644
--- a/node_modules/@vue/reactivity/dist/reactivity.d.ts
+++ b/node_modules/@vue/reactivity/dist/reactivity.d.ts
@@ -1,5 +1,5 @@
 
-declare type BaseTypes = string | number | boolean;
+declare type BaseTypes = string | number | boolean | Element | Document | Window;
 
 declare type CollectionTypes = IterableCollections | WeakCollections;
 
