diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.d.ts b/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
index 409b622..d198ddb 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
@@ -237,7 +237,7 @@ export declare type ComponentObjectPropsOptions<P = Data> = {
 
 export declare type ComponentOptions = ComponentOptionsWithoutProps<any, any, any, any, any> | ComponentOptionsWithObjectProps<any, any, any, any, any> | ComponentOptionsWithArrayProps<any, any, any, any, any>;
 
-declare interface ComponentOptionsBase<Props, RawBindings, D, C extends ComputedOptions, M extends MethodOptions, E extends EmitsOptions, EE extends string = string> extends LegacyOptions<Props, D, C, M>, SFCInternalOptions, ComponentCustomOptions {
+export declare interface ComponentOptionsBase<Props, RawBindings, D, C extends ComputedOptions, M extends MethodOptions, E extends EmitsOptions, EE extends string = string> extends LegacyOptions<Props, D, C, M>, SFCInternalOptions, ComponentCustomOptions {
     setup?: (this: void, props: Props, ctx: SetupContext<E>) => RawBindings | RenderFunction | void;
     name?: string;
     template?: string | object;
